### ISSUE TITLE
Feature Request - Option to save N,W,S,E camera

### ISSUE THREAD

**Issue by G-lint** [1Defence/teleport-zoom#1]
Firstly, thank you for making such an awesome plugin. Its been extremely useful in certain cases for AFKing and Skilling :)
I would love a separate set of hotkeys that saves and then applies any of the right-clickable compass options.  

Forgive me as I'm someone that is completely lost around code so I don't know how hard/annoying something like this would be to implement but I would love even it it had to be 4 separate possible hotkeys say:

- Alt+N+PageUp - Save zoom + Change compass to North
- Alt+E+PageUp - Save zoom + Change compass to East
- Alt+S+PageUp - Save zoom + Change compass to South
- Alt+W+PageUp - Save zoom + Change compass to West

**Comment by 1Defence** [comment]
Yeah directional saves is something I've wanted to add, haven't got around to it because I'm indecisive on how I'd have the user set it. I was trying to avoid having excessive hotkeys registered but at any rate I'll revisit at some point
